Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -1065,6 +1065,10 @@ private void LoadTemplates(string path)
 
         public string GetFullPath(string filePath)
         {
+            // if the path is root of a drive in windows, which normally returns C:, return C:\ instead.
+            if (filePath.Length==2 && filePath[0].ToString().ToUpper()[0] >='A' && filePath[0].ToString().ToUpper()[0] <='Z' && Path.DirectorySeparatorChar=='\\' && filePath[filePath.Length - 1] == ':')
```

Comment:
You don't need to check if the first character is within upper letter range, you just need to check if its not `/` so we ensure that this is not a Unix path. Also your code right now does not work with `c:`.